### ISSUE TITLE
meson 0.64.0 ModuleNotFoundError

### ISSUE THREAD

**Issue by kiufta** [mesonbuild/meson#11100]
**Describe the bug**
```
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/mesonbuild/mesonmain.py", line 194, in run
    return options.run_func(options)
  File "/usr/lib/python3.10/site-packages/mesonbuild/minstall.py", line 759, in run
    b = build.load(opts.wd)
  File "/usr/lib/python3.10/site-packages/mesonbuild/build.py", line 2907, in load
    obj = pickle.load(f)
ModuleNotFoundError: No module named 'mesonbuild.mesonlib.universal'; 'mesonbuild.mesonlib' is not a package
```

**To Reproduce**
https://gitlab.gnome.org/GNOME/gtk/-/blob/gtk-3-24/meson.build

**Expected behavior**
`DESTDIR="$pkgdir" meson install -C build` runs through

**system parameters**
* ArchLinux
* Python 3.10.8
* meson 0.64.0

**Comment by eli-schwartz** [comment]
Duplicate of #11051

**Comment by eli-schwartz** [comment]
Note that the fix was released 34 minutes ago in meson 0.64.1 :)

**Comment by kiufta** [comment]
> Note that the fix was released 34 minutes ago in meson 0.64.1 :)

Please forgive me my 22 minutes lag. I'm over 30 ;)

**Comment by eli-schwartz** [comment]
I would never blame someone for not realizing a bug got fixed in between discovering the bug and reporting it :p which is a strange coincidence, and not at all obvious to users that they should check *again* for a new version.
